contrib/coredns/policy/policy.go

```diff
@@ -140,6 +142,12 @@ func (p *policyPlugin) parseOption(c *caddy.Controller) error {
 
 	case "transfer":
 		return p.parseTransfer(c)
+
```

Comment:
I think it's time to update README with all new options.

Reply:
Updated